package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend-icons",
-  "version": "1.8.0",
+  "version": "1.8.1",
```

Comment:
it is not patch version... it is adding new icon

Reply:
Well the version has to change, but not to 1.8.1 but you have to raise minor version...
you are adding new stuff, please read about https://semver.org

also now package-lock is not in sync